<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="4" total="6" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="40731" started-at="2017-04-02T13:10:48Z" finished-at="2017-04-02T13:11:28Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="40731" started-at="2017-04-02T13:10:48Z" finished-at="2017-04-02T13:11:28Z">
      <class name="org.xframium.examples.java.test.WebHome">
        <test-method status="PASS" signature="setupSuite(org.testng.ITestContext)[pri:0, instance:org.xframium.examples.java.test.WebHome@3dd4520b]" name="setupSuite" is-config="true" duration-ms="1743" started-at="2017-04-02T09:10:46Z" finished-at="2017-04-02T09:10:48Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@3a4621bd]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupSuite -->
        <test-method status="PASS" signature="beforeMethod(java.lang.reflect.Method, [Ljava.lang.Object;, org.testng.ITestContext)[pri:0, instance:org.xframium.examples.java.test.WebHome@3dd4520b]" name="beforeMethod" is-config="true" duration-ms="3415" started-at="2017-04-02T09:10:48Z" finished-at="2017-04-02T09:10:51Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void org.xframium.examples.java.test.WebHome.testNativeSeleniumAppium(org.xframium.device.ng.TestContainer) throws java.lang.Exception]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[[Ljava.lang.Object;@31dadd46]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[org.testng.TestRunner@3a4621bd]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="PASS" signature="beforeMethod(java.lang.reflect.Method, [Ljava.lang.Object;, org.testng.ITestContext)[pri:0, instance:org.xframium.examples.java.test.WebHome@3dd4520b]" name="beforeMethod" is-config="true" duration-ms="9217" started-at="2017-04-02T09:10:48Z" finished-at="2017-04-02T09:10:57Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void org.xframium.examples.java.test.WebHome.testNativeSeleniumAppium(org.xframium.device.ng.TestContainer) throws java.lang.Exception]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[[Ljava.lang.Object;@4ed5eb72]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[org.testng.TestRunner@3a4621bd]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="FAIL" signature="testNativeSeleniumAppium(org.xframium.device.ng.TestContainer)[pri:0, instance:org.xframium.examples.java.test.WebHome@3dd4520b]" name="testNativeSeleniumAppium" duration-ms="114" started-at="2017-04-02T09:10:51Z" data-provider="deviceManager" finished-at="2017-04-02T09:10:51Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.xframium.device.ng.TestContainer@359a0a16]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:86)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at org.junit.Assert.assertTrue(Assert.java:52)
	at org.xframium.examples.java.test.WebHome.testNativeSeleniumAppium(WebHome.java:249)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:696)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:882)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:75)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:14)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testNativeSeleniumAppium -->
        <test-method status="PASS" signature="afterMethod(java.lang.reflect.Method, [Ljava.lang.Object;, org.testng.ITestResult, org.testng.ITestContext)[pri:0, instance:org.xframium.examples.java.test.WebHome@3dd4520b]" name="afterMethod" is-config="true" duration-ms="1492" started-at="2017-04-02T09:10:51Z" finished-at="2017-04-02T09:10:53Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void org.xframium.examples.java.test.WebHome.testNativeSeleniumAppium(org.xframium.device.ng.TestContainer) throws java.lang.Exception]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[[Ljava.lang.Object;@31dadd46]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[[TestResult name=testNativeSeleniumAppium status=FAILURE method=WebHome.testNativeSeleniumAppium(org.xframium.device.ng.TestContainer)[pri:0, instance:org.xframium.examples.java.test.WebHome@3dd4520b] output={null}]]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[org.testng.TestRunner@3a4621bd]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="FAIL" signature="testNativeSeleniumAppium(org.xframium.device.ng.TestContainer)[pri:0, instance:org.xframium.examples.java.test.WebHome@3dd4520b]" name="testNativeSeleniumAppium" duration-ms="872" started-at="2017-04-02T09:10:57Z" data-provider="deviceManager" finished-at="2017-04-02T09:10:58Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.xframium.device.ng.TestContainer@359a0a16]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:86)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at org.junit.Assert.assertTrue(Assert.java:52)
	at org.xframium.examples.java.test.WebHome.testNativeSeleniumAppium(WebHome.java:249)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:696)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:882)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:75)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:14)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testNativeSeleniumAppium -->
        <test-method status="PASS" signature="afterMethod(java.lang.reflect.Method, [Ljava.lang.Object;, org.testng.ITestResult, org.testng.ITestContext)[pri:0, instance:org.xframium.examples.java.test.WebHome@3dd4520b]" name="afterMethod" is-config="true" duration-ms="2790" started-at="2017-04-02T09:10:58Z" finished-at="2017-04-02T09:11:01Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void org.xframium.examples.java.test.WebHome.testNativeSeleniumAppium(org.xframium.device.ng.TestContainer) throws java.lang.Exception]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[[Ljava.lang.Object;@4ed5eb72]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[[TestResult name=testNativeSeleniumAppium status=FAILURE method=WebHome.testNativeSeleniumAppium(org.xframium.device.ng.TestContainer)[pri:0, instance:org.xframium.examples.java.test.WebHome@3dd4520b] output={null}]]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[org.testng.TestRunner@3a4621bd]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="beforeMethod(java.lang.reflect.Method, [Ljava.lang.Object;, org.testng.ITestContext)[pri:0, instance:org.xframium.examples.java.test.WebHome@3dd4520b]" name="beforeMethod" is-config="true" duration-ms="7921" started-at="2017-04-02T09:11:01Z" finished-at="2017-04-02T09:11:08Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void org.xframium.examples.java.test.WebHome.testXFramiumWithJavaPOM(org.xframium.device.ng.TestContainer) throws java.lang.Exception]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[[Ljava.lang.Object;@19b93fa8]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[org.testng.TestRunner@3a4621bd]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="PASS" signature="beforeMethod(java.lang.reflect.Method, [Ljava.lang.Object;, org.testng.ITestContext)[pri:0, instance:org.xframium.examples.java.test.WebHome@3dd4520b]" name="beforeMethod" is-config="true" duration-ms="3310" started-at="2017-04-02T09:11:01Z" finished-at="2017-04-02T09:11:04Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void org.xframium.examples.java.test.WebHome.testXFramiumWithJavaPOM(org.xframium.device.ng.TestContainer) throws java.lang.Exception]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[[Ljava.lang.Object;@12f9af83]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[org.testng.TestRunner@3a4621bd]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="FAIL" signature="testXFramiumWithJavaPOM(org.xframium.device.ng.TestContainer)[pri:0, instance:org.xframium.examples.java.test.WebHome@3dd4520b]" name="testXFramiumWithJavaPOM" duration-ms="577" started-at="2017-04-02T09:11:04Z" data-provider="deviceManager" finished-at="2017-04-02T09:11:04Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.xframium.device.ng.TestContainer@6950f47f]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:86)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at org.junit.Assert.assertTrue(Assert.java:52)
	at org.xframium.examples.java.test.WebHome.testXFramiumWithJavaPOM(WebHome.java:151)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:696)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:882)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:75)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:14)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testXFramiumWithJavaPOM -->
        <test-method status="PASS" signature="afterMethod(java.lang.reflect.Method, [Ljava.lang.Object;, org.testng.ITestResult, org.testng.ITestContext)[pri:0, instance:org.xframium.examples.java.test.WebHome@3dd4520b]" name="afterMethod" is-config="true" duration-ms="1445" started-at="2017-04-02T09:11:04Z" finished-at="2017-04-02T09:11:06Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void org.xframium.examples.java.test.WebHome.testXFramiumWithJavaPOM(org.xframium.device.ng.TestContainer) throws java.lang.Exception]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[[Ljava.lang.Object;@12f9af83]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[[TestResult name=testXFramiumWithJavaPOM status=FAILURE method=WebHome.testXFramiumWithJavaPOM(org.xframium.device.ng.TestContainer)[pri:0, instance:org.xframium.examples.java.test.WebHome@3dd4520b] output={null}]]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[org.testng.TestRunner@3a4621bd]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="FAIL" signature="testXFramiumWithJavaPOM(org.xframium.device.ng.TestContainer)[pri:0, instance:org.xframium.examples.java.test.WebHome@3dd4520b]" name="testXFramiumWithJavaPOM" duration-ms="1865" started-at="2017-04-02T09:11:08Z" data-provider="deviceManager" finished-at="2017-04-02T09:11:10Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.xframium.device.ng.TestContainer@6950f47f]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:86)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at org.junit.Assert.assertTrue(Assert.java:52)
	at org.xframium.examples.java.test.WebHome.testXFramiumWithJavaPOM(WebHome.java:151)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:696)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:882)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:75)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:14)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testXFramiumWithJavaPOM -->
        <test-method status="PASS" signature="afterMethod(java.lang.reflect.Method, [Ljava.lang.Object;, org.testng.ITestResult, org.testng.ITestContext)[pri:0, instance:org.xframium.examples.java.test.WebHome@3dd4520b]" name="afterMethod" is-config="true" duration-ms="2761" started-at="2017-04-02T09:11:10Z" finished-at="2017-04-02T09:11:13Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void org.xframium.examples.java.test.WebHome.testXFramiumWithJavaPOM(org.xframium.device.ng.TestContainer) throws java.lang.Exception]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[[Ljava.lang.Object;@19b93fa8]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[[TestResult name=testXFramiumWithJavaPOM status=FAILURE method=WebHome.testXFramiumWithJavaPOM(org.xframium.device.ng.TestContainer)[pri:0, instance:org.xframium.examples.java.test.WebHome@3dd4520b] output={null}]]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[org.testng.TestRunner@3a4621bd]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="beforeMethod(java.lang.reflect.Method, [Ljava.lang.Object;, org.testng.ITestContext)[pri:0, instance:org.xframium.examples.java.test.WebHome@3dd4520b]" name="beforeMethod" is-config="true" duration-ms="7953" started-at="2017-04-02T09:11:13Z" finished-at="2017-04-02T09:11:21Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void org.xframium.examples.java.test.WebHome.testXFramiumWithKWPOM(org.xframium.device.ng.TestContainer) throws java.lang.Exception]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[[Ljava.lang.Object;@7e6ef134]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[org.testng.TestRunner@3a4621bd]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="PASS" signature="beforeMethod(java.lang.reflect.Method, [Ljava.lang.Object;, org.testng.ITestContext)[pri:0, instance:org.xframium.examples.java.test.WebHome@3dd4520b]" name="beforeMethod" is-config="true" duration-ms="3198" started-at="2017-04-02T09:11:13Z" finished-at="2017-04-02T09:11:16Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void org.xframium.examples.java.test.WebHome.testXFramiumWithKWPOM(org.xframium.device.ng.TestContainer) throws java.lang.Exception]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[[Ljava.lang.Object;@1f010bf0]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[org.testng.TestRunner@3a4621bd]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="PASS" signature="testXFramiumWithKWPOM(org.xframium.device.ng.TestContainer)[pri:0, instance:org.xframium.examples.java.test.WebHome@3dd4520b]" name="testXFramiumWithKWPOM" duration-ms="1895" started-at="2017-04-02T09:11:16Z" data-provider="deviceManager" finished-at="2017-04-02T09:11:18Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.xframium.device.ng.TestContainer@ad985ef]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testXFramiumWithKWPOM -->
        <test-method status="PASS" signature="afterMethod(java.lang.reflect.Method, [Ljava.lang.Object;, org.testng.ITestResult, org.testng.ITestContext)[pri:0, instance:org.xframium.examples.java.test.WebHome@3dd4520b]" name="afterMethod" is-config="true" duration-ms="1223" started-at="2017-04-02T09:11:18Z" finished-at="2017-04-02T09:11:19Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void org.xframium.examples.java.test.WebHome.testXFramiumWithKWPOM(org.xframium.device.ng.TestContainer) throws java.lang.Exception]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[[Ljava.lang.Object;@1f010bf0]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[[TestResult name=testXFramiumWithKWPOM status=SUCCESS method=WebHome.testXFramiumWithKWPOM(org.xframium.device.ng.TestContainer)[pri:0, instance:org.xframium.examples.java.test.WebHome@3dd4520b] output={null}]]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[org.testng.TestRunner@3a4621bd]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="testXFramiumWithKWPOM(org.xframium.device.ng.TestContainer)[pri:0, instance:org.xframium.examples.java.test.WebHome@3dd4520b]" name="testXFramiumWithKWPOM" duration-ms="5388" started-at="2017-04-02T09:11:21Z" data-provider="deviceManager" finished-at="2017-04-02T09:11:26Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.xframium.device.ng.TestContainer@ad985ef]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testXFramiumWithKWPOM -->
        <test-method status="PASS" signature="afterMethod(java.lang.reflect.Method, [Ljava.lang.Object;, org.testng.ITestResult, org.testng.ITestContext)[pri:0, instance:org.xframium.examples.java.test.WebHome@3dd4520b]" name="afterMethod" is-config="true" duration-ms="1935" started-at="2017-04-02T09:11:26Z" finished-at="2017-04-02T09:11:28Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void org.xframium.examples.java.test.WebHome.testXFramiumWithKWPOM(org.xframium.device.ng.TestContainer) throws java.lang.Exception]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[[Ljava.lang.Object;@7e6ef134]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[[TestResult name=testXFramiumWithKWPOM status=SUCCESS method=WebHome.testXFramiumWithKWPOM(org.xframium.device.ng.TestContainer)[pri:0, instance:org.xframium.examples.java.test.WebHome@3dd4520b] output={null}]]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[org.testng.TestRunner@3a4621bd]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="afterSuite()[pri:0, instance:org.xframium.examples.java.test.WebHome@3dd4520b]" name="afterSuite" is-config="true" duration-ms="208" started-at="2017-04-02T09:11:28Z" finished-at="2017-04-02T09:11:29Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterSuite -->
      </class> <!-- org.xframium.examples.java.test.WebHome -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
